grunt/esbuild/build.js

```diff
@@ -43,6 +43,7 @@ const nodeConfig = {
   platform: 'node',
   entryPoints: ['src/platform/nodejs/index.ts'],
   outfile: 'build/ably-node.js',
+  external: ['ws', 'got'],
```

Comment:
I think it would be beneficial to add a comment here explaining why we want to do this (based on that [slack convo](https://ably-real-time.slack.com/archives/C030C5YLY/p1711542833244969?thread_ts=1711541241.653079&cid=C030C5YLY)).
We already had 'ws' and 'got' marked as externals in v1: https://github.com/ably/ably-js/blob/1.2.50/webpack.config.js#L60-L61 when we were building node using webpack (I assume that's why v1 works in bun), but we lost this config when migrated to esbuild.
With a comment explaining what issues it fixes it would be less likely we remove it accidentally.

Reply:
good idea, i've added a comment explaining this now 👍

Reply:
great!